src/main/java/facades/WeatherFacade.java

```diff
@@ -53,18 +57,25 @@ public static WeatherFacade getWeatherFacade() {
                     while (scan.hasNext()) {
                         response += scan.nextLine();
                     }
-                    JsonArray array = GSON.fromJson(response, JsonArray.class);
-                    array.forEach(object -> {
-                        weatherReports.add(GSON.fromJson(object, WeatherDTO.class));
-                    });
+                    JsonParser jsonParser = new JsonParser();
+                    JsonElement jsonElement = jsonParser.parse(response);
+                    if (jsonElement.isJsonObject()) {
+                        errorMessage = GSON.fromJson(response, JsonObject.class).toString();
+                    } else if (jsonElement.isJsonArray()) {
+                        JsonArray array = GSON.fromJson(response, JsonArray.class);
+                        array.forEach(object -> {
+                            weatherReports.add(GSON.fromJson(object, WeatherDTO.class));
+                        });
+                    }
```

Comment:
If it is an Object, that means it is the error object. 
`{
  "code": 400,
  "message": "Requested city could not be found"
}`
If it is an array, that means it is the data we need.

Reply:
This didn't work apparently. 
Could use some help fixing this issue tomorrow. 
Error Message should be one of these two, depending on endpoint:
![image](https://user-images.githubusercontent.com/11192971/69829798-c82f8680-1221-11ea-8b94-30409a149e4d.png)

Reply:
Fixed in 73a126d146afc07f717a66a50ceaba9403d4070b